coral-component-masonry/src/scripts/MasonryItem.js

```diff
@@ -239,6 +275,18 @@ const MasonryItem = Decorator(class extends BaseComponent(HTMLElement) {
       masonry._onItemDisconnected(this);
     }
   }
+
+  /**
+   @class Coral.Masonry.Item.AccessibilityState
+   @classdesc The Masonry Item Accessibility State
+   @htmltag coral-masonry-item-accessibilitystate
+   @extends HTMLElement
+   */
+  accessibilityState() {
+    const element = document.createElement('coral-masonry-item-accessibilitystate');
+    element.className = 'u-coral-screenReaderOnly';
+    return element;
```

Comment:
Please move `Coral.Masonry.Item.AccessibilityState` to new file  and export from there.  I also think to export it as Coral.Masonry.Item.AccessibilityState you need to update here : https://github.com/adobe/coral-spectrum/blob/master/coral-component-masonry/index.js#L36